application/laravel/README.md

```diff
@@ -0,0 +1,33 @@
+## Prerequisites
```

Comment:
For consistency we should add an introduction to the readme.

Reply:
Fixed!